term-keys.el

```diff
@@ -48,98 +48,98 @@ file."
-    ["<menu>"		"Menu"		127	"Menu"		#x0010	nil		nil		"MENU"		"Apps"		nil	nil	nil	]
-    [nil		"Control_R"	97	"Ctrl"		nil	nil		nil		"RIGHT_CONTROL"	"RControl"	nil	nil	nil	]
-
-    ["<up>"		"Up"		103	"Up"		#xF700	nil		nil		"UP"		"Up"		nil	nil	nil	]
-    ["<down>"		"Down"		108	"Down"		#xF701	nil		nil		"DOWN"		"Down"		nil	nil	nil	]
-    ["<left>"		"Left"		105	"Left"		#xF702	nil		nil		"LEFT"		"Left"		nil	nil	nil	]
-    ["<right>"		"Right"		106	"Right"		#xF703	nil		nil		"RIGHT"		"Right"		nil	nil	nil	]
-
-    ["<insert>"		"Insert"	110	"Ins"		#xF746	nil		nil		"INSERT"	"Insert"	nil	nil	nil	]
-    ["<delete>"		"Delete"	111	"Del"		#xF728	nil		nil		"DELETE"	"Delete"	nil	nil	nil	]
-    ["<home>"		"Home"		102	"Home"		#xF729	nil		nil		"HOME"		"Home"		nil	nil	nil	]
-    ["<end>"		"End"		107	"End"		#xF72B	nil		nil		"END"		"End"		nil	nil	nil	]
-    ["<prior>"		"Prior"		104	"PgUp"		#xF72C	nil		nil		"PAGE_UP"	"PageUp"	nil	nil	nil	]
-    ["<next>"		"Next"		109	"PgDown"	#xF72D	nil		nil		"PAGE_DOWN"	"PageDown"	nil	nil	nil	]
+  ;; Emacs		X11		TTY	Qt		macOS	Emacs shifted	X11 shifted	GLFW		winit		Qt sh.	macOS-S	macOS-C Wezterm
+
+  '(["<escape>"		"Escape"	1	"Esc"		#x001B	nil		nil		"ESCAPE"	"Escape"	nil	nil	#x001b	"Escape"	]
+    ["<f1>"		"F1"		59	"F1"		#xF704	nil		nil		"F1"		"F1"		nil	nil	nil	"F1"		]
+    ["<f2>"		"F2"		60	"F2"		#xF705	nil		nil		"F2"		"F2"		nil	nil	nil	"F2"		]
+    ["<f3>"		"F3"		61	"F3"		#xF706	nil		nil		"F3"		"F3"		nil	nil	nil	"F3"		]
+    ["<f4>"		"F4"		62	"F4"		#xF707	nil		nil		"F4"		"F4"		nil	nil	nil	"F4"		]
+    ["<f5>"		"F5"		63	"F5"		#xF708	nil		nil		"F5"		"F5"		nil	nil	nil	"F5"		]
+    ["<f6>"		"F6"		64	"F6"		#xF709	nil		nil		"F6"		"F6"		nil	nil	nil	"F6"		]
+    ["<f7>"		"F7"		65	"F7"		#xF70A	nil		nil		"F7"		"F7"		nil	nil	nil	"F7"		]
+    ["<f8>"		"F8"		66	"F8"		#xF70B	nil		nil		"F8"		"F8"		nil	nil	nil	"F8"		]
+    ["<f9>"		"F9"		67	"F9"		#xF70C	nil		nil		"F9"		"F9"		nil	nil	nil	"F9"		]
+    ["<f10>"		"F10"		68	"F10"		#xF70D	nil		nil		"F10"		"F10"		nil	nil	nil	"F10"		]
+    ["<f11>"		"F11"		87	"F11"		#xF70E	nil		nil		"F11"		"F11"		nil	nil	nil	"F11"		]
+    ["<f12>"		"F12"		88	"F12"		#xF70F	nil		nil		"F12"		"F12"		nil	nil	nil	"F12"		]
+    ["<print>"		"Print"		99	"Print"		#xF710	nil		nil		"PRINT_SCREEN"	"Snapshot"	nil	nil	nil	"Print"		]
+    ["<Scroll_Lock>"	"Scroll_Lock"	70	"ScrollLock"	nil	nil		nil		"SCROLL_LOCK"	"Scroll"	nil	nil	nil	"ScrollLock"	]
+    ["<pause>"		"Pause"		119	"Pause"		nil	nil		nil		"PAUSE"		"Pause"		nil	nil	nil	"Pause"		]
+
+    ["`"		"grave"		43	"`"		?`	"~"		"asciitilde"	"GRAVE_ACCENT"	"Grave"		"~"	?~	nil	"`"		]
+    ["1"		"1"		2	"1"		?1	"!"		"exclam"	"1"		"Key1"		"!"	?!	nil	"1"		]
+    ["2"		"2"		3	"2"		?2	"@"		"at"		"2"		"Key2"		"@"	?@	nil	"2"		]
+    ["3"		"3"		4	"3"		?3	"#"		"numbersign"	"3"		"Key3"		"#"	?#	nil	"3"		]
+    ["4"		"4"		5	"4"		?4	"$"		"dollar"	"4"		"Key4"		"$"	?$	nil	"4"		]
+    ["5"		"5"		6	"5"		?5	"%"		"percent"	"5"		"Key5"		"%"	?%	nil	"5"		]
```

Comment:
Do shifted sequences work, like C-S-5? If not then this needs another column.

Reply:
I think you are right. `C-h k` when I press that reports "% runs the command self-insert-command". I have now added a shift column and emacs says it sees it as `C-%` -- is that what you would expect, or have I done something wrong?

Reply:
That looks right!